src/security/tests/cli_security_tests.c

```diff
@@ -303,6 +306,21 @@ test_request_credentials_fails_if_reply_token_missing(void **state)
 	daos_iov_free(&creds);
 }
 
+static void
+test_request_credentials_fails_if_reply_verifier_missing(void **state)
+{
+	d_iov_t creds;
+
+	memset(&creds, 0, sizeof(d_iov_t));
+	auth__token__free_unpacked(drpc_call_resp_return_auth_credential->verifier,
```

Comment:
(style)  line over 80 characters

Reply:
Could probably shorten the existing name if you wanted. `drpc_call_resp_auth_cred`, maybe?